pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/CancerStore.java

```diff
@@ -56,4 +65,67 @@ public Set<CancerItem> getCancerList(String languageIsoCode) {
         }
     }
 
+    /**
```

Comment:
Brought these methods over from `GetCancerSuggestionsRoute` and refactored them a bit.